src/app.ts

```diff
@@ -84,6 +85,8 @@ function createWindow(): void {
     mainWindow.maximize()
   }
 
+  setAppMenuBarVisibility()
```

Comment:
Is this something we can set on the window when creating it with the default value?  Then we can call this function later?

Reply:
> When the menu is turned off, how do you access it?

As mentioned in the OP, we can hit the Alt key once to bring up the Menu bar. And to keep users informed, this message below will be shown when the User hides the menu bar each time.

![image](https://user-images.githubusercontent.com/12471103/65961875-1ec45280-e475-11e9-8ebf-cd5fca0f3567.png)

This will be shown each time the Menu bar is hidden by the user. Should we show this message only once?

Reply:
> Is this something we can set on the window when creating it with the default value? Then we can call this function later?

Yes, we can set the `autoHideMenuBar` property while creating the window but a separate call to `mainWindow.setMenuBarVisibility(boolean)` is required to make the Alt key work. Without making this call, the Menu bar visibility will be set as per the config value but hitting the Alt key will not bring it up.